mobify/platform-scaffold
# Fix PDP variations dropdown on SFCC Connector

# Fix PDP variation selection

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-67

## Changes
- When switching between variations on a PDP, the user will no longer be scrolled to the top of the PDP
- Stores the selected variation URL in the app state, and instead of using `browserHistory.push()` to navigate to a new variant, `fetchPdpData` is called to refresh the data on the PDP
- Adds a default option in the `<select/>` element of variant dropdowns
- This PR fixes the correct variants not being selected when clicking `Edit Cart`

## How to test-drive this PR
- Use `demandware-connector`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate to a PDP with multiple variants (Womens -> Lace trimmed cardigan)
- Select options, and ensure the page does not scroll to the top
- Add to cart, and proceed to edit cart, ensuring that the proper variant is prepopulated
